topeft/modules/dataDrivenEstimation.py

```diff
@@ -118,7 +118,9 @@ def DDFakes(self):
                         hPromptSub = hPromptSub.remove("systematic", syst_var_idet_rm_lst)
 
                         # now we actually make the subtraction
-                        hPromptSub.scale(-1)
+                        # var(A - B) = var(A) + var(B)
+                        if '_sumw2' not in key:
```

Comment:
maybe change this check to be: `key.endswith("_sumw2")` so that it's more explicit that this is checking for a postfix addition.

Reply:
Done